### DISCUSSION TITLE
Stepper component - Vertical items in horizontal orientation

### DISCUSSION THREAD

**Discussion by mnakhli** [mantinedev/mantine#5197] [Feature requests]
Add another mode to the stepper component like this [image] I made. The icon, title and description are arranged vertically, but the overall state of this component remains horizontal.

<img width="848" alt="Screenshot 1402-08-15 at 11 05 36" src="https://github.com/mantinedev/mantine/assets/21153535/ed79feb6-fabb-45d8-8b89-9313e07d9245">

**Comment by bzitzow-osh** [comment]
+1

**Comment by bzitzow-osh** [comment]
@mnakhli were you able to extend the component as is to support this implementation?
